src/nodedef.h

```diff
@@ -469,6 +480,11 @@ struct ContentFeatures
 	bool isLiquid() const{
 		return (liquid_type != LIQUID_NONE);
 	}
+	bool hasLiquidMovePhysics() const{
+		return (liquid_move_physics == LIQUID_MOVE_LIQUID) ||
+			(liquid_move_physics == LIQUID_MOVE_AUTO &&
+			liquid_type != LIQUID_NONE);
```

Comment:
Wouldn't it make more sense to do the auto -> liquid or not decision in `read_content_features()`?